File: Level 0/코딩 기초 트레이닝/전국 대회 선발 고사/solution.py
def solution(ranks, attendances):
    students = sorted(
        [
            (rank, i)
            for i, (rank, attendance) in enumerate(zip(ranks, attendances))
            if attendance
        ]
    )
    a, b, c = map(lambda x: x[1], students[:3])
    return 10000 * a + 100 * b + c

File: Level 0/코딩 기초 트레이닝/전국 대회 선발 고사/solution.js
function solution(ranks, attendances) {
  const [a, b, c] = ranks
    .map((rank, index) => [index, rank])
    .filter((_, index) => attendances[index])
    .sort((a, b) => a[1] - b[1])
    .map(([student]) => student);

  return 10000 * a + 100 * b + c;
}
